src/sentry/integrations/jira/utils.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import absolute_import
+
+import hashlib
+import jwt
+
+from six.moves.urllib.parse import quote
+
+from sentry.models import Integration
+
+
+class JiraValidationError(Exception):
+    pass
+
+
+def percent_encode(val):
+    # see https://en.wikipedia.org/wiki/Percent-encoding
+    return quote(val.encode('utf8', errors='replace')).replace('%7E', '~').replace('/', '%2F')
```

Comment:
Does this belong here in `integrations.jira.utils`? Is it some special way of percent encoding

Reply:
do you mean here vs `sentry.utils`? i can move it there

Reply:
Yeah exactly.